README.md

```diff
@@ -118,14 +119,15 @@ The task **gulp build** will be run automatically.
   src/                    --> source files for the application
     hello.js              --> added just for an example
   test/                   --> test files for the application
-    .jshintrc             --> configuration file for jshint; these rules will 
+    .jshintrc             --> configuration file for jshint; these rules will
                               be applied for files in this folder only;
                               created because test cases contain global functions
                               which exports by mocha(describe, it,beforeEach, etc)
     hello.test.js         --> added just for an example
   .jscsrc                 --> configuration file for jscs
   .jshintrc               --> configuration file for jshint
   .travis.yml             --> configuration file for travis-ci
+  .editorconfig           --> configuration file for code editors to keep style
```

Comment:
Could you please move line with editor config before .jscsrc, since the list is sorted alphabetically :)